class BaseError(Exception):
    pass


class IncludeError(BaseError):
    pass
